### ISSUE TITLE
Mosaic attenuation pattern in lung

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6559]
**Preferred term label:** Mosaic attenuation pattern in lung

**Synonyms** Mosaic perfusion

**Definition (free text, please give PubMed ID)**
Mosaic attenuation refers to heterogeneous areas of differing pulmonary attenuation on CT imaging. Mosaic attenuation may result from occlusive vascular disease, obstructive small airways disease, pulmonary oedema or pulmonary haemorrhage PMID: 26274445

**Parent term (use hpo.jax.org/app)**
Pulmonary interstitial high-resolution computed tomography abnormality HP:0025389

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
adding as Mosaic pulmonary attenuation pattern
